exp/api/v1alpha3/azuremanagedcontrolplane_conversion.go

```diff
@@ -81,3 +82,8 @@ func (dst *AzureManagedControlPlaneList) ConvertFrom(srcRaw conversion.Hub) erro
 	src := srcRaw.(*infrav1exp.AzureManagedControlPlaneList)
 	return Convert_v1beta1_AzureManagedControlPlaneList_To_v1alpha3_AzureManagedControlPlaneList(src, dst, nil)
 }
+
+// Convert_v1beta1_ManagedControlPlaneSubnet_To_v1alpha3_ManagedControlPlaneSubnet is an autogenerated conversion function.
+func Convert_v1beta1_ManagedControlPlaneSubnet_To_v1alpha3_ManagedControlPlaneSubnet(in *infrav1exp.ManagedControlPlaneSubnet, out *ManagedControlPlaneSubnet, s apiconversion.Scope) error {
```

Comment:
I know this is wrong, but I'm not sure how to do it right. Without it, it gives

```
E0907 12:05:50.092587    9028 conversion.go:756]   the following fields need manual conversion:
E0907 12:05:50.092592    9028 conversion.go:758]       - ServiceEndpoints
```

but with it, it's a duplicate declaration when trying to build the app.